pubspec.yaml

```diff
@@ -1,26 +1,30 @@
 name: dart2js_info
-version: 0.5.9
+version: 0.5.10
+
 description: >
   Libraries and tools to process data produced when running dart2js with
   --dump-info.
 author: Dart Team <misc@dartlang.org>
 homepage: https://github.com/dart-lang/dart2js_info/
 
 environment:
-  sdk: '>=2.0.0-dev.17.0 <2.0.0'
+  sdk: '>=2.0.0-dev.17.0 <3.0.0'
+
 dependencies:
-  args: '>=0.13.0 <2.0.0'
+  args: ^1.4.3
   charcode: ^1.1.0
   collection: ^1.10.1
   fixnum: ^0.10.5
   path: ^1.3.6
   protobuf: ^0.9.0
-  quiver: '>=0.21.0 <0.26.0'
-  shelf: '>=0.6.1+2 <0.8.0'
+  quiver: ^0.29.0
+  shelf: ^0.7.3
```

Comment:
Same question for `quiver` and `shelf`

Reply:
See above

Reply:
Yes, I understand the general idea, but this package can now only be used under Dart 2 (since the min SDK is 2-dev).

The smallest version of `shelf` that is Dart 2 compatible is `^0.7.3`.
The same remark applies to quiver.

Reply:
My understanding is that pub will consider the sdk compatibility as well during version solving, so as long as you have a version range that includes a Dart2 compatible version, pub should find that.

Reply:
Agreed, but then the range is misleading.